demo/runners/alice.py

```diff
@@ -229,32 +230,41 @@ async def main(
 
     try:
         log_status("#7 Provision an agent and wallet, get back configuration details")
-        agent = AliceAgent(
-            start_port,
-            start_port + 1,
-            genesis_data=genesis,
-            no_auto=no_auto,
-            timing=show_timing,
-            multitenant=multitenant,
-        )
-        await agent.listen_webhooks(start_port + 2)
-
-        with log_timer("Startup duration:"):
-            await agent.start_process()
-        log_msg("Admin URL is at:", agent.admin_url)
-        log_msg("Endpoint URL is at:", agent.endpoint)
+        if multitenant:
+            base_start_port = BASE_AGENT_PORT
+            # Note that Alice acts as a just controller (use admin/endpoint of BaseAgent)
+            agent = AliceAgent(
+                "Alice",
+                base_start_port,
+                base_start_port + 1,
+                multitenant=multitenant
+            )
+            await agent.listen_webhooks(start_port + 2)
+            log_msg("Controller URL of Alice is at:", agent.webhook_url)
+            await agent.create_wallet_and_did()
+        else:
+            agent = AliceAgent(
+                "Alice.Agent",
+                start_port,
+                start_port + 1,
+                genesis_data=genesis,
+                no_auto=no_auto,
+                timing=show_timing,
+            )
+            await agent.listen_webhooks(start_port + 2)
+            with log_timer("Startup duration:"):
+                await agent.start_process()
+            log_msg("Admin URL is at:", agent.admin_url)
+            log_msg("Endpoint URL is at:", agent.endpoint)
 
         log_status("#9 Input faber.py invitation details")
         await input_invitation(agent)
 
         options = (
             "    (3) Send Message\n"
             "    (4) Input New Invitation\n"
-        )
-        if multitenant:
-            options += "    (W) Create and/or Enable Wallet\n"
```

Comment:
See comment from above, I'd prefer to keep these options in the demo

Reply:
Restored the options.